gulpfile.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-/* global process, __dirname */
+/* eslint-env node */
```

Comment:
This is a cleaner/simpler way to tell ESLint that a file runs under Node and should have access to all of Node's global variables.